src/python/WMComponent/TaskArchiver/CleanCouchPoller.py

```diff
@@ -161,8 +161,9 @@ def algorithm(self, parameters=None):
         try:
             logging.info("Creating workload summary")
             finishedwfsWithLogCollectAndCleanUp = DataCache.getFinishedWorkflows()
+            logging.info("Total %s workflow summay will be created", len(finishedwfsWithLogCollectAndCleanUp))
             self.archiveSummaryAndPublishToDashBoard(finishedwfsWithLogCollectAndCleanUp)
-            logging.info("%s workload summary is created", len(finishedwfsWithLogCollectAndCleanUp))
+            logging.info("Workload summary is created")
```

Comment:
isn't it 'workflow summary'?

Reply:
I am not sure we can go with workflow summary. :-) I will updated the patch

Reply:
actually it seems we are using workload summary fot the name.

Reply:
then workload summary it is ;)